src/Environment/Environment.php

```diff
@@ -258,6 +265,9 @@ public function addExtension(ExtensionInterface $extension): EnvironmentBuilderI
 
     private function initializeExtensions(): void
     {
+        // Initialize the slug normalizer
+        $this->getSlugNormalizer();
```

Comment:
I know this is needed because it needs to add a `DocumentRenderedEvent` listener, I just don't think we need to retrieve something that may not actually be used.

Perhaps there should be a listener on the `Environment` instead that does something like the following:
```php
    public function onDocumentRendered(DocumentRenderedEvent $event): void
    {
        // Immediately return if there is no slug normalizer to proxy event to.
        if ($this->slugNormalizer === null) {
            return;
        }

        this->slugNormalizer->onDocumentRendered($event);
    }
```